README.md

```diff
@@ -256,6 +256,7 @@ IpForwarding      | target\_instance\_ips  | `false` disables internal IP addres
 MetadataScripts   | run\_dir               | String base directory where metadata scripts are executed.
 MetadataScripts   | startup                | `false` disables startup script execution.
 MetadataScripts   | shutdown               | `false` disables shutdown script execution.
+MetadataScripts   | default\_shell         | String with the default shell to execute the scripts.
```

Comment:
Please alphabetize - also `s/the scripts/scripts`

Reply:
Done